### ISSUE TITLE
Is it possible to show title on top of URL?

### ISSUE THREAD

**Issue by TheMithuRoy** [NickM-27/LinkPreview#35]
Is it possible to display the og:title on top of the URL? Instead of showing YouTube Video, the video title may show up. And instead of displaying the full link, just the hostname would be really nice.

**Comment by lazy-coder-10** [comment]
Yes it is possible. you i have to import library and make changes in LinkPreview.kt file.

**Comment by NickM-27** [comment]
If you make this change and do a pr I'd be happy to accept it

**Comment by lazy-coder-10** [comment]
Okay, I will try if I got the time to do but in the meantime if someone is not able to fetch the youtube title you just need to remove the below code and the else condition code will be outside the else branch. so that means jsoup will fetch the title and image from the youtube URL as well as the normal url.

` if (url.let { it.contains("youtube") && it.contains("v=") }) {
                  val id = url.split("v=")[1].split(" ")[0]
                  val imageUrl = "https://img.youtube.com/vi/$id/hqdefault.jpg"
                  imageType = ImageType.YOUTUBE
                  context.addLink(url, imageUrl)
                  setPreviewData(PreviewData("Youtube Video",imageUrl,url))

              }`
